draft-ietf-rats-corim.md

```diff
@@ -2489,6 +2500,17 @@ The profile must specify how to compare the CBOR tagged Reference Value against
 
 Note that a Verifier may compare Reference Values in any order, so the comparison should not be stateful.
 
+## Concise Software Identifier Extension {#sec-coswid}
+
+A CoRIM may contain a CoSWID tag as specified in {{-coswid}}.
+The CoRIM profile allows for the specifier to provide meaning to extension points in the "Specification Required" range of the extension points `$version-scheme` and `$$coswid-extension`.
+The CoRIM profile is not permitted to specify negative values for these extension points.
+To allow for tag-level scoping of profile identifier to bundle multiple items under a single CoRIM, this specification registers a `$$coswid-extension` to specifically name the profile that gives meaning to the profile-defined extensions.
```

Comment:
```suggestion
This specification registers an extension to `concise-swid-tag` using the `$$coswid-extension` mechanism. This extension defines a CoSWID profile code point that allows for an optional CoSWID profile identifier.
The CoSWID profile identifier takes precedence over the CoRIM profile identifier.

* `profile-override` (index 58): A profile identifier that overrides the `corim-map.profile` to interpret the contents of the `concise-swid-tag`.
The `corim-map.profile` MUST NOT change the interpretation of the `CoSWID` tag when a `profile-override` value is supplied.
```

Reply:
Since this registration is to CoSWID Items, I specifically avoided "profile-override". A CoSWID could be created on its own with the "profile" code point as a reference in itself to the specification for any extension on the base spec.

Reply:
I kind of think "profile" is better overall since the spec language describes scoping semantics normatively. Other specs could incorporate CoMID or CoBOM subsets resulting in a similar dynamic as with CoSWID.

Reply:
I shall update the other occurrences as well then.